packages/meta-updater/src/updateTSConfig.ts

```diff
@@ -89,14 +89,18 @@ export async function updateTSConfig(
     references.push({ path: relativePath });
   }
 
+  const compilerOptions = {
+    ...(cloneDeep(input.compilerOptions) ?? {}),
+  };
+  delete compilerOptions.outDir;
+  delete compilerOptions.rootDir;
+
   return {
     ...input,
     extends: getExtends(pathFromPackageToRoot, input.extends),
-    compilerOptions: {
-      ...(input.compilerOptions ?? {}),
-      rootDir: "src",
-      outDir: "out",
-    },
+
+    ...(isEqual(compilerOptions, {}) ? {} : { compilerOptions }),
```

Comment:
mixed feelings. Is it better to leave the empty `compilerOptions: {}`?

Reply:
I could go either way.